src/transitivity/transitivity.jl

```diff
@@ -0,0 +1,30 @@
+"""
+    ```transitiveclosure!(g::SimpleGraph, selflooped = false)```
+Compute the transitive closure of an undirected graph.
+If `selflooped` is true, add self loops to the graph.
+
+### Implementation Notes
+This version of the function modifies the original graph.
+"""
+function transitiveclosure!(g::Graph, selflooped = false)
```

Comment:
Maybe we should use `@traitfn` here - similar how we do it for directed graphs at the moment.

Reply:
Sure!